package.json

```diff
@@ -2,7 +2,7 @@
     "name": "mindaro",
     "displayName": "Bridge to Kubernetes",
     "description": "Rapid Kubernetes development for teams",
-    "version": "2.0.120230310",
+    "version": "2.0.120230321-vscodedependentboot",
```

Comment:
💡☕️ Thanks for this, please note: I do not see any update on the `package.json` file yet the `package-lock` file has many. Essentially, `package-lock` file is the automatically generated file. here is more for read: https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json

Reply:
Ah this was leftover from testing. Need to remove it, thanks.